tox.ini

```diff
@@ -115,7 +115,7 @@ description = Run pre-commit on staged files or arbitrary pre-commit commands (t
 commands = pre-commit {posargs:run}
 
 [docs]  # Sphinx documentation configuration
-extras = docs
+extras = full, docs
```

Comment:
With the inclusion of `"canonical-sphinx[full]"` in `docs` is this change still necessary?

Reply:
Good catch, this change is now unnecessary

Reply:
Resolved with 5ab6bb5